tutorial_code/example2_3_buckyball_orbits.py

```diff
@@ -14,70 +14,72 @@
 #
 from os.path import exists
 
-import dimod
+import matplotlib
 import networkx as nx
 import numpy as np
+import matplotlib.pyplot as plt
 
-from dwave.system import DWaveSampler
-from matplotlib import pyplot as plt
+import dimod
 
 from helpers import orbits
-from helpers.embedding_helpers import raster_embedding_search
 from helpers.orbits import get_orbits
 
 
-def main(visualize=True):
-    """Main function to run example
+def main():
+    """Reproduces Figure 11 of the shimming tutorial DOI:10.3389/fcomp.2023.1238988
 
-    Args:
-        visualize (bool, optional): flag for visualization. Defaults to True.
+    Example of creating a Binary Quadratic Model (BQM) structured as a
+    buckyball (Buckminsterfullerene) and computing its orbits (symmetries of
+    spins and couplers).
     """
+
     # Parse the BQM
     path_to_csv = "data/bucky_ball.csv"
     if not exists(path_to_csv):
         path_to_csv = f"tutorial_code/{path_to_csv}"
-    J = {(int(e[0]), int(e[1])): w for *e, w in np.loadtxt(path_to_csv, delimiter=",")}
+    if not exists(path_to_csv):
+        raise FileNotFoundError(
+            f"CSV file not found in specified paths. Checked: 'data/bucky_ball.csv' and '{path_to_csv}'"
+        )
+    try:
+        J = {
+            (int(e[0]), int(e[1])): w
+            for *e, w in np.loadtxt(path_to_csv, delimiter=",")
+        }
+    except Exception as e:
+        raise ValueError(f"Failed to load or parse CSV file '{path_to_csv}': {e}")
```

Comment:
Never good to simply except `Exception`. Just leaving it without a try-except block should raise a `FileNotFound` error printing the path, so maybe just remove that here?